exporter/tanzuobservabilityexporter/transformer_test.go

```diff
@@ -294,6 +294,75 @@ func TestSpanForSourceTag(t *testing.T) {
 	assert.Equal(t, "source_from_span_attribute", actual.Tags["_source"])
 }
 
+func TestSpanForDroppedEventsCount(t *testing.T) {
+	inNanos := int64(50000000)
```

Comment:
I think you could combine these three tests into one.  starting on line 310, you'd have 3 assert.NotContains(), one for each of the new tags.  One line 313, you would call 3 different Set methods on span: SetDroppedEventsCount, SetDroppedLinksCount, and SetDroppedAttributesCount. Finally you would have 3 assert.Equal statements starting on line 17.  If you decide to combine the three tests into one, remember to rename the test.

Reply:
done